package.json

```diff
@@ -60,9 +60,9 @@
     "pelias-logger": "0.2.0",
     "pelias-microservice-wrapper": "1.1.3",
     "pelias-model": "5.0.0",
-    "pelias-query": "8.16.1",
+    "pelias-query": "pelias/query#9673b57",
     "pelias-sorting": "1.0.1",
-    "pelias-text-analyzer": "1.8.3",
+    "pelias-text-analyzer": "pelias/text-analyzer#91196dd",
```

Comment:
what is this convention? does the hash refer to github? I thought the domain name was required?

don't forget to remove these before merging!